more/problem-sets-competitive-programming.md

```diff
@@ -34,6 +33,7 @@
 * [DMOJ](https://dmoj.ca)
 * [E-olymp](https://www.e-olymp.com/en/)
 * [Exercism](https://exercism.org)
+* [GeeksforGeeks](https://practice.geeksforgeeks.org)
```

Comment:
we have 7 geeksforgeeks links already, do we need another?

Reply:
Sorry! I didn't check if it was available on other files. As it was not mentioned in the practice set problems, so i thought of adding it.

Reply:
Shall i edit by removing the gfg link as i have also deleted a source (binary search) which doesn't seem to exist?

Reply:
yes, please work on this branch